projectile.el

```diff
@@ -729,6 +729,16 @@ position."
           (const :tag "Move to end" move-to-end)
           (const :tag "Keep" keep)))
 
+(defcustom projectile-limit-project-buffer-count nil
```

Comment:
I think we don't really need this option. If the other one is `nil` we can consider this as no limit.

Reply:
Sure, I just pushed a version without this custom.